### PR TITLE
Send analog button values based on W3C GamePad

### PR DESCRIPTION
**Pull request by mbhatt627**

### CODE REVIEW

**Comment by mbhatt627** [comment]
This PR dependent on  lib WPE changes for getting Gamepad button values  from 00 to 1.0. 
https://github.com/WebPlatformForEmbedded/libwpe/pull/133

https://www.w3.org/TR/gamepad/#dom-gamepadbutton-value

value attribute
For buttons that have an analog sensor, this property MUST represent the amount which the button has been pressed. All button values MUST be linearly normalized to the range [0.0 .. 1.0]. 0.0 MUST mean fully unpressed, and 1.0 MUST mean fully pressed. For buttons without an analog sensor, only the values **0.0 and 1.0** for fully unpressed and fully pressed respectively, MUST be provided.

File: Source/WebCore/platform/gamepad/wpe/WPEGamepad.cpp [hunk 1]
```diff
@@ -84,6 +89,15 @@ void WPEGamepad::absoluteAxisChanged(unsigned axis, double value)
     WPEGamepadProvider::singleton().scheduleInputNotification(*this, WPEGamepadProvider::ShouldMakeGamepadsVisible::Yes);
 }
 
+void WPEGamepad::analogButtonChanged(unsigned button, double value)
+{
+    m_lastUpdateTime = MonotonicTime::now();
+    m_buttonValues[button].setValue(value);
```
Comment (ceyusa) [inline Source/WebCore/platform/gamepad/wpe/WPEGamepad.cpp:95]: can we assure that the lower layers will send values between 0 and 1? 

would it make sense to clamp or even assert those received values?

**Comment by ceyusa** [comment]
this issue should be closed in favor of #1442

**Comment by pgorszkowski-igalia** [comment]
https://github.com/WebPlatformForEmbedded/WPEWebKit/pull/1442 is the newest implementation of this PR. I close this PR.

**Pull request closed without merging**